libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -187,6 +187,9 @@ string ABIFunctions::requestedFunctions()
 
 string ABIFunctions::cleanupFunction(Type const& _type, bool _revertOnFailure)
 {
+	if (_type.category() == Type::Category::Address)
```

Comment:
I think it would be better to call the integer cleanup function inside the address cleanup function to keep the naming scheme, i.e. generate `function cleanup_revert_t_address(value) -> cleaned { cleaned := cleanup_revert_t_uint160(value) }`

Reply:
The same is done for contracts, for example.